README.md

```diff
@@ -53,7 +53,7 @@ Pre-built packages for Windows, macOS, and Linux are found on the
 Install from source
 -------------------
 
-See the [Building Neovim](https://github.com/neovim/neovim/wiki/Building-Neovim) wiki page for details.
+See the [Building Neovim](https://github.com/neovim/neovim/wiki/Building-Neovim) wiki page and [Support](https://neovim.io/doc/user/support.html) for details.
```

Comment:
I would like to keep the explicit link to the list of supported platforms here.

Reply:
I think I did what you meant. If not just suggest the change you want.